docs/source/roles/researcher/user_guide.md

```diff
@@ -481,6 +481,12 @@ Once connected to it, you can analyse the sensitive data belonging to your proje
   :align: center
   ```
 
+````{note}
+The Linux desktops within our SRDs use the [Ubuntu operating system](https://ubuntu.com/).
+The desktop environment used by our SRDs is called [Xfce](https://www.xfce.org/).
+For more detailed information on how to use Xfce, consult the official guidance on the [Xfce website](https://docs.xfce.org/xfce/getting-started)
```

Comment:
https://docs.xfce.org/xfce/xfdesktop/usage might be a better link for Xfce usage.
We should try and avoid docs about installation, configuration.

Reply:
I looks like "user docs" are alongside "technical docs" so there might not be a great solution.
I think Xfce is fairly intuitive though, so not a big problem.

Reply:
I've changed it so the link goes directly to the "Desktop Environment" section, which links out to probably the most relevant sections, and added more specific links for the desktop and window managers